backend.native/cli.bc/src/org/jetbrains/kotlin/cli/bc/K2NativeCompilerArguments.kt

```diff
@@ -152,11 +152,13 @@ class K2NativeCompilerArguments : CommonCompilerArguments() {
     )
     var frameworkImportHeaders: Array<String>? = null
 
-    // FIXME: same name.
     @Argument(
-            value = "-Xg0",
+            value = "-Xadd-light-debug",
             valueDescription = "{disable|enable}",
-            description = "Add light debug information"
+            deprecatedName = "-Xg0",
```

Comment:
The idea is to avoid using `-Xg0` at all. Otherwise existing projects adding `-Xg0` (without value) will fail to compile.